core/UserManager/UserHandler.py

```diff
@@ -73,4 +73,47 @@ def handler_user_update(request_data, user):
             user_serializer.save()
             return Response(user_serializer.data, status=status.HTTP_202_ACCEPTED)
         else:
-            return Response(user_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(user_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    @staticmethod
+    def handler_get_user_info(user):
+        """
+        Handler for getting user info.
+
+        :param user_id:
+        :return: dict with user info - first name, last name, email,address, phone number
+        """
+        try:
+            serialized_user = UserInfoSerializer(user)
+
+            if serialized_user:
+                return Response(serialized_user.data, status=status.HTTP_200_OK)
+        except Exception as e:
+            return Response({'user_error': "Something went wrong, please try again later or contact support.",
+                                "dev_error": str(e)}, status=status.HTTP_400_BAD_REQUEST)
+            
+        return Response({'user_error': "No user was found."}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+    @staticmethod
+    def handler_user_change_password(request_data, user):
+        """
+        Handler for changing user password
+
+        :param user:
+        :param request_data:
+        :return: success message
+        """
+
+        if not "password" in request_data or request_data.get('password') is None:
+            return Response("Password must be passed in and cannot be null or empty", status = status.HTTP_400_BAD_REQUEST)
+
+        UserHelper.validate_password(request_data.get('password'))
+
+        if user:
+            user.set_password(request_data.get('password'))
+
+            return Response("Password has been updated successfully.", status=status.HTTP_200_OK)
+        else:
+            return Response(validated_data.errors, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Getting and errror on `validated_data.errors`, maybe you meant to use another variable.